avalon/pipeline.py

```diff
@@ -248,6 +248,56 @@ def process(self, session, **kwargs):
         pass
 
 
+class InventoryAction(object):
+    """A custom action for the scene inventory tool
+
+    If registered the action will be visible in the Right Mouse Button menu
+    under the submenu "Actions".
+
+    """
+
+    label = None
+    icon = None
+    color = None
+    order = 0
+
+    @staticmethod
+    def is_compatible(container):
+        """Override function in a custom class
+
+        This method is specifically used to ensure the action can operate on
+        the container.
+
+        Args:
+            container: dict
```

Comment:
This should actually be `container (dict): container` or something like that. I believe with google docstrings the type needs to be between the braces.